app/services/vacancies/export/dwp_find_a_job/published_and_updated_vacancies/parsed_vacancy.rb

```diff
@@ -67,6 +68,8 @@ def type_id
         TYPE_PERMANENT_ID
       when "fixed_term", "parental_leave_cover"
         TYPE_CONTRACT_ID
+      when "casual"
+        TYPE_TEMPORARY_ID
```

Comment:
I quite like the pattern of looking this sort of thing up in a hash with fetch - thus resulting in a sentry alert if we add a value but forget to update the mapping. WDYT?

Reply:
@starswan I was wondering how to keep track of future changes and if we could do some metaprogramming with the tests iterating over all the defined values in the Vacancy enums/constants... 
But I like the simplicity of the fetch approach :clap: